identity/src/main/java/com/android/identity/CredentialData.java

```diff
@@ -1159,7 +1160,8 @@ Collection<X509Certificate> getAuthKeysNeedingCertification() {
                 try {
                     // Calculate name to use and be careful to avoid collisions when
                     // re-certifying an already populated slot.
-                    String aliasForAuthKey = mCredentialKeyAlias + String.format("_auth_%d", n);
+                    String aliasForAuthKey = mCredentialKeyAlias + String.format(Locale.US,
```

Comment:
Using locale here is actually bad, because we want the same value around the world.

This is deriving the name of a key and it needs to be globally consistent.

Reply:
The current code does not use "the same value around the world". The current code will use different values if the user changes the language setting on their phone.

Reply:
Ah, yes. I see that now. You fixed it to US. :facepalm: